router/dataplane.go

```diff
@@ -1033,14 +1059,16 @@ type scionPacketProcessor struct {
 	// For a hop performing an Xover, it is the MAC corresponding to the down segment.
 	cachedMac []byte
 	// macBuffers avoid allocating memory during processing.
-	macBuffers macBuffers
+	// (VerifiedSCION) Having the same name for the type and the field caused an exception
+	// with Gobra so the type name was slightly altered
+	macBuffers macBuffersT
 
 	// bfdLayer is reusable buffer for parsing BFD messages
 	bfdLayer layers.BFD
 }
 
-// macBuffers are preallocated buffers for the in- and outputs of MAC functions.
-type macBuffers struct {
+// macBuffersT are preallocated buffers for the in- and outputs of MAC functions.
```

Comment:
```suggestion
// macBuffersT are preallocated buffers for the in- and outputs of MAC functions.
// (VerifiedSCION) This type used to be called macBuffers but this lead to an exception in
// Gobra because there is a field with name and type macBuffers. Because of that, we renamed it.
```